tango/__main__.py

```diff
@@ -26,12 +26,14 @@
       --log-level [debug|info|warning|error]
                                       Set the global log level.
       --no-logging                    Disable logging altogether.
-      --file-friendly-logging         Outputs progress bar status on separate lines and slows refresh rate.
+      --file-friendly-logging         Outputs progress bar status on
+                                      separate lines and slows refresh rate.
       --help                          Show this message and exit.
 
     Commands:
-      info  Get info about the current tango installation.
-      run   Run a tango experiment.
+      info    Get info about the current tango installation
+      run     Run a tango experiment
+      server  Run a local webserver that watches a workspace
```

Comment:
Should also add a blurb below about this command, like we have for `tango run` and `tango info`.

Reply:
44f79ec2b58efa9bec063c475e761009db3ceccf